app/js/controllers/actions/event/event-list-ctrl.js

```diff
@@ -0,0 +1,299 @@
+'use strict';
```

Comment:
double check that the indentation in this file matches the other files. This seems to be 4-space, but the others are 2-space iirc?